Cargo.toml

```diff
@@ -68,6 +68,7 @@ opendal = {version = "0.43.0", features = ["layers-prometheus"]}
 signal-hook = { version = "0.3.17", features = ["iterator"]}
 signal-hook-tokio = { version = "0.3.1", features = ["futures-v0_3"] }
 tokio-util = "0.7.10"
+siphasher = "0.3.1"
```

Comment:
Better not introduce new dependency, unless it's a "must".
There are too many hasher out there, need throughout investigate to pick one